.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+### Before submitting
+
+Please complete the following checklist when submitting a PR:
+
+- [ ] All new features must include a unit test.
+      If you've fixed a bug or added code that should be tested, add a test to the
+      [`tests/`](tests/) directory!
```

Comment:
Nice, I didn't know you could do links like this! But it doesn't work, does it need a `../tests/`? This is the URL it sends to:
`https://github.com/PennyLaneAI/pennylane-lightning/blob/repo_guidelines/.github/tests`

Reply:
Good catch! I forgot to add a sufficient number of `../`. I'll update this to be absolute to the `tests` dir, rather than relative.

Reply:
On second thought, the relative path you suggested is much tidier.